docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -446,9 +444,95 @@ Where,
 Iron Fortification
 ~~~~~~~~~~~~~~~~~~
 
+Population Coverage Data and Coverage Algorithm
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The baseline coverage for iron fortification is the same as the baseline 
+coverage for folic acid fortification, as described below_.
+
+.. note:: 
+
+	A limitation of our population coverage data is that we assume no baseline 
+	iron fortification occurs in the absence of folic acid fortification and 
+	vice versa.
+
+Additionally, the coverage algorithm should be implemented in the same way as 
+for folic acid fortification, as described here_.
+
+Effect Size
+^^^^^^^^^^^
+
+Iron fortification of staple food affects two outcomes in our simulation 
+model. The first outcome is an individual's hemoglobin concentration following 
+the *direct* consumption of iron fortified foods. The second outcome is a 
+simulant's birth weight following the *maternal* consumption of iron fortified 
+foods.
+
+**Hemoglobin Level**
```

Comment:
Is there a reason for just using bold text instead of another heading level?

Reply:
I didn't see another option for another subheading level in the RST documentation guide I was working off of. If you know of one feel free to directly edit.